tests/decorators/redux.js

```diff
@@ -1,5 +0,0 @@
-@connect(mapStateToProps, mapDispatchToProps)
-export class MyApp extends React.Component {}
-
-@connect(state => ({ todos: state.todos }))
-export class Home extends React.Component {}
```

Comment:
Can you keep all those tests somehow and fix them? They are real piece of code that broke in the past, I want to make sure that we keep them as regression test.